src/com/nutrons/framework/util/FlowOperators.java

```diff
@@ -37,11 +47,15 @@
    * @return the motor input values with values smaller than 0.2 removed
    */
   public static Flowable<Double> deadband(Flowable<Double> input) {
-    return input.map((x) -> abs(x) < 0.2 ? 0.0 : x);
+    return input.map((x) -> abs(x) < 0.4 ? 0.0 : x);
+  }
+
+  public static io.reactivex.functions.Function<Double, Double> deadbandMap(double minimum, double maximum, double remap) {
+    return bandMap(minimum, maximum, x -> remap);
   }
 
-  public static Function<Double, Double> deadbandMap(Flowable<Double> input) {
-    return x -> abs(x) < 0.2 ? 0.0 : x;
+  public static io.reactivex.functions.Function<Double, Double> bandMap(double minimum, double maximum, Function<Double, Double> remap) {
```

Comment:
We were supposed to change the interface of `deadband`.  Is this in a different branch?